conda/environments/raft_dev_cuda11.5.yml

```diff
@@ -23,6 +23,7 @@ dependencies:
 - doxygen>=1.8.20
 - libfaiss>=1.7.0
 - faiss-proc=*=cuda
+- ccache
```

Comment:
Shall we do the same changes in cuml? I personally always use `cuml_dev` environment for raft development, and I suspect that's not me only.

Reply:
Agree with you. If this PR gets accepted by ops team, we could do the same in cuML too.